app/routes/endemics/test-urn.js

```diff
@@ -34,7 +34,7 @@ const previousPageUrl = (request) => {
   if (typeOfReview === claimType.review) return `${urlPrefix}/${endemicsVetRCVS}`
   if (typeOfReview === claimType.endemics) {
     if (latestVetVisitApplication) {
-      if ([livestockTypes.beef, livestockTypes.dairy].includes(typeOfLivestock)) return `${urlPrefix}/${endemicsVetVisitsReviewTestResults}`
+      if (typeOfLivestock === livestockTypes.dairy) return `${urlPrefix}/${endemicsVetVisitsReviewTestResults}`
```

Comment:
You can actually remove this whole line because dairy won't get here either - it is being removed for follow-up

Reply:
Thanks, Will remove it